src/utils/constants.ts

```diff
@@ -5,6 +5,7 @@ import {
 type OperationTypeEnum = components['schemas']['OperationTypeEnum'];
 type StatusTypeEnum = components['schemas']['Key1d2Enum'];
 type ProgrammeTypeEnum = components['schemas']['Key1d2Enum'];
+type DrefStatus = components['schemas']['OperationTypeEnum'];
```

Comment:
This is duplicated.

Reply:
Is this a mistake.

Reply:
Might need to create a separate enum in the server